### PR TITLE
Map25: resending the slideout, which had conflicts

### PR DESCRIPTION
**Pull request by telemaxx**

### CODE REVIEW

**Comment by telemaxx** [comment]
i had again the conflicts. so tried eg the flowing deletion via browser:
grafik

can you check if it is removed in this PR?

File: bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs [hunk 1]
```diff
@@ -125,15 +159,26 @@ sp_cleanup.remove_trailing_whitespaces_ignore_empty=false
 sp_cleanup.remove_unnecessary_array_creation=false
 sp_cleanup.remove_unnecessary_casts=true
 sp_cleanup.remove_unnecessary_nls_tags=true
-sp_cleanup.remove_unused_imports=true
+sp_cleanup.remove_unused_imports=false
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/.settings/org.eclipse.jdt.ui.prefs:162]: Can you fix this file, this morning I spend time to revert your deletion and now it is modified again. 
This value `sp_cleanup.remove_unused_imports=true` cannot be `false`

All of these files must be the same

mt ui prefs

**Comment by telemaxx** [comment]
now its:
sp_cleanup.remove_unused_imports=true

**Comment by wolfgang-ch** [comment]
> now its:
> sp_cleanup.remove_unused_imports=true

OK, this value is set correctly but you have saved it propably with Eclipse 2021-03 which added a lot of new properties. As I pointed out above, **ALL** `.settings` **MUST** be the same, now they are not.

Can you revert this file back to the original value which I already did this morning.

mt ui prefs

**Pull request merged**